internal/pkg/mail/sendmail.go

```diff
@@ -0,0 +1,52 @@
+package mail
+
+import (
+	"bytes"
+	"github.com/TheFlies/ofriends/internal/app/types"
+	envconfig "github.com/TheFlies/ofriends/internal/pkg/config/env"
+	"github.com/go-gomail/gomail"
+	"github.com/pkg/errors"
+	"html/template"
+)
+
+type EmailConfig struct {
+	OutGoingMailServer string `envconfig:"OUT_GOING_MAIL_SERVER" default:"smtp.tma.com.vn"`
+	MailServerPort     int    `envconfig:"OUT_GOING_MAIL_PORT" default:"25"`
+	Subject            string `envconfig:"EMAIL_SUBJECT" default:"Prepare for customer visit"`
+	MailFrom           string `envconfig:"MAIL_FORM" default:"ofriends.oficial@tma.com.vn"`
+	MailTo             string `envconfig:"MAIL_TO" default:"pdkhang@tma.com.vn"` // for test, to avoiding  send real to internal communication department
+}
+
+func SendEmail(data []types.NotificationData) error {
+	var conf EmailConfig
+	envconfig.Load(&conf)
+
+	dataForMail := types.VisitsData{
+		Data: data,
+	}
+
+	htmlTemplate, err := template.ParseFiles("internal/pkg/mail/emailtemplate.html")
```

Comment:
Because template never change, you can just parse it once at package init and keep in the global vars instead of parsing every time like this.